noop.go

```diff
@@ -60,3 +60,7 @@ func (*NoopSpan) IsEntry() bool {
 func (*NoopSpan) IsExit() bool {
 	return false
 }
+
+func (*NoopSpan) IsValid() bool {
+	return false
```

Comment:
I think it is better to keep no-op span always valid? The codes would expect a span valid in most cases, right?

Reply:
The reason I think about this is, if `return false` always, we will face a case `a new created span` is `Invalid`, which could cause some unexpected bugs switching from tracing to noop mode.

Reply:
Yeah it's true, I thought about it for a while, the only side-effect is the no-op span is always be valid, even after callling `End`, I will modify it